litmus-portal/frontend/src/components/Header/ProfileDropdownItems.tsx

```diff
@@ -1,54 +1,65 @@
 /* eslint-disable react/no-array-index-key */
 import React, { useState } from 'react';
-import { withStyles, createStyles, Theme } from '@material-ui/core/styles';
 import {
   Popover,
   Typography,
   Avatar,
   Divider,
   Button,
-  Badge,
   List,
   ListItem,
   ListItemText,
 } from '@material-ui/core';
-import CameraAltOutlinedIcon from '@material-ui/icons/CameraAltOutlined';
-import ProjectListItem from '../ProjectListItem';
+import ProjectListItem from './ProjectListItem';
 import useStyles from './styles';
 import * as UserActions from '../../redux/actions/user';
 import { history } from '../../redux/configureStore';
 import useActions from '../../redux/actions';
+import { Project, ProjectsCallBackType } from '../../models/header';
 
-const SmallAvatar = withStyles((theme: Theme) =>
-  createStyles({
-    root: {
-      width: 24,
-      height: 24,
-      border: `3px solid ${theme.palette.background.paper}`,
-    },
-  })
-)(Avatar);
-
-interface ProfileInfoDropdownProps {
-  anchorEl: any;
-  isOpen: any;
-  onClose: any;
-  name: any;
-  email: any;
-  projects: any;
+interface OnCloseType {
+  (): void;
```

Comment:
OnCloseType only has one param of an anonymous function. If you want to keep it that way, its fine but give it some name, like `func: () => void ;` but since you only have one and you are calling it on the `ProfileInfoDropdownItemProps` interface, it makes more sense to just add `onClose: () => void;` right there